pkg/gce-pd-csi-driver/controller.go

```diff
@@ -872,6 +878,24 @@ func (gceCS *GCEControllerServer) DeleteVolume(ctx context.Context, req *csi.Del
 	return gceCS.deleteSingleDeviceDisk(ctx, req, project, volKey)
 }
 
+func getHostNameFromPVC(ctx context.Context, pvcName, pvcNamespace string, kubeClient *kubernetes.Clientset) string {
+	if kubeClient == nil {
+		// Client is initialized only when location hint is enabled
+		return ""
+	}
+
+	pvc, err := kubeClient.CoreV1().PersistentVolumeClaims(pvcNamespace).Get(ctx, pvcName, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("Failed to get persistent volume claims: %v", err)
+		// Don't log the error as create volume request can be continued without hostname
+		return ""
+	}
+	if hostName, ok := pvc.Annotations[annSelectedNode]; ok {
+		return hostName
```

Comment:
Add some logging when we found the selected node -- it will make it easier to confirm that things are working as expected.

Reply:
Added.